### DISCUSSION TITLE
View Permissions Tab?

### DISCUSSION THREAD

**Discussion by CrypticCanadian** [JustOptimize/return-ShowHiddenChannels#37] [Ideas]
Heyo, appreciate tf outta you for keeping this up. 

Is there any plan to re-add the View Channel Permissions tab? Showing which roles/members have access to view each channel? 

Cheers!

**Comment by JustOptimize** [comment]
Yes, I plan to restore all functionality.
(It will take some time because I have a lot of school and non-school work that is a higher priority).

**Comment by JustOptimize** [comment]
And yes, I called the permission tab "right click menu thingy" in the readme as you can see [here](https://user-images.githubusercontent.com/54294419/196409634-4c59e2e4-ebbe-4aad-a047-cc90b32005c4.jpg)
